collection/screenshot/capture-screenshot-via-keybd-event.yml

```diff
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: capture screenshot via keybd event
+    namespace: collection/screenshot
+    authors:
+      - "@_re_fox"
+    scope: function
+    att&ck:
+      - Collection::Screen Capture [T1113]
+    mbc:
+      - Collection::Screen Capture [E1113]
+    examples:
+      - 3f3bbcf8fd90bdcdcdc5494314ed4225:0x402D10
+  features:
+    - and:
+      - basic block:
+        - and:
+          - operand[0].number: 0x2C = VK_SNAPSHOT
+          - count(api(user32.keybd_event)): 2
+          - or:
+            - operand[0].number: 0x3 #KEYEVENTF_KEYUP|KEYEVENTF_EXTENDEDKEY
+            - operand[0].number: 0x2 #KEYEVENTF_KEYUP
```

Comment:
```suggestion
            - operand[0].number: 0x3 = KEYEVENTF_KEYUP|KEYEVENTF_EXTENDEDKEY
            - operand[0].number: 0x2 = KEYEVENTF_KEYUP
```

Reply:
Thanks for the suggestion, I'll commit the changes